client/extensions/woocommerce/state/selectors/plugins.js

```diff
@@ -10,27 +10,51 @@ import config from 'config';
 import { getPlugins, isRequestingForSites } from 'state/plugins/installed/selectors';
 import { getRequiredPluginsForCalypso } from 'woocommerce/lib/get-required-plugins';
 import { getSelectedSiteId } from 'state/ui/selectors';
+import createSelector from 'lib/create-selector';
+
+const getWcsPluginData = createSelector(
```

Comment:
Rather than define a new selector, can we just use [`getPluginOnSite()`](https://github.com/Automattic/wp-calypso/blob/master/client/state/plugins/installed/selectors.js#L97)?

Reply:
The primary reason is that I didn't know it existed. I found a couple of things:
* `getPluginOnSite()` isn't memoized and it does a non-trivial amount of work.
* `getPluginOnSite()` can't filter by `active`, and the returned data structure is something like `{ version: '1.2.3', sites: { 357532: { active: true } } }`

I've refactored the new selector to use `getPluginOnSite()` but I kept it to keep the memoization and a consistent interface (returns `null` if the plugin is not active). Is that better?